File: Daily Flash/week 4/day 2/MySolutions/Python/Program2.py
print("\n  Binary to Decimal\n")

num=int(input("\nEnter a binary number : "))
ans=0
n=1
temp=0

for i in range(num+1):
	temp=num %10
	num= num//10
	
	if(temp == 1):
		ans=ans+n
	
	n=n*2
	
print("\nDecimal number : ",ans)
print()			

'''
Output :

(base) mayuri@mayuri-PC:~/C2WT/Daily Flash/week 4/day 2/MySolutions/Python$ python3 Program2.py

  Binary to Decimal


Enter a binary number : 1111

Decimal number :  15

(base) mayuri@mayuri-PC:~/C2WT/Daily Flash/week 4/day 2/MySolutions/Python$ python3 Program2.py

  Binary to Decimal


Enter a binary number : 101

Decimal number :  5
'''

File: Daily Flash/week 4/day 2/MySolutions/C/Program2.c
/*
Problem Statement :

Program 2: Write a Program to Convert entered Binary Number to Decimal
Number
Input: Binary Number: 1011
Output: Decimal Number: 11

*/

#include<stdio.h>

void main() {

	int num;
	int ans=0,temp,n=1;
	
	printf("\nEnter binary number :: ");
	scanf("%d",&num);
	
	while(num) {
	
		temp=num%10;
		num=num/10;
		
		if(temp == 1)
			ans=ans+n;
			
		n=n*2;		
	}
	
	printf("\nDecimal number is :  %d\n\n",ans);

}


/*
Output : 

(base) mayuri@mayuri-PC:~/C2WT/Daily Flash/week 4/day 2/MySolutions/C$ cc Program2.c
(base) mayuri@mayuri-PC:~/C2WT/Daily Flash/week 4/day 2/MySolutions/C$ ./a.out

Enter binary number :: 1011

Decimal number is :  11

(base) mayuri@mayuri-PC:~/C2WT/Daily Flash/week 4/day 2/MySolutions/C$ ./a.out

Enter binary number :: 11010

Decimal number is :  26



*/
